index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="../styles.css">
+    <title>Intention Timer</title>
+  </head>
+  <body>
+    <header class="intention-timer">
+      <h1 class="heading">Intention Timer</h1>
+    </header>
+    <main>
+      <section class="new-activity">
+        <h2>New Activity</h2>
+        <section class="user-input">
+          <label for="activity-list">Select a category:</label>
```

Comment:
May need to add some more containers around this section to ensure we can get the layout right.